design/use_cases/multi-story-building_use_case.md

```diff
@@ -0,0 +1,15 @@
+# Multi-story Building Navigation (2D+)
+As a Robot user I want my robot to be able to navigate stairways, ramps or elevators to move to another portion of the multi-story building so that it can do something useful
```

Comment:
'multi-story building' do that mean having several maps and switching from one another ? Or a single one ? How does that operate with the localization system ?

Reply:
@orduno - I would expect the robot to be able to know to navigate into the elevator, at which point it might provide a message to another module which then controls the elevator and tells it when to get off. I believe there will be other "navigate to a known point, broadcast X, listen for Y, proceed to next location" scenarios.